src/pipecat/adapters/schemas/function_schema.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+from typing import Any, Dict, List
+
+
+class FunctionSchema:
+    def __init__(
+        self, name: str, description: str, properties: Dict[str, Any], required: List[str]
+    ) -> None:
+        """Standardized function schema representation.
+
+        :param name: Name of the function.
+        :param description: Description of the function.
+        :param properties: Dictionary defining properties types and descriptions.
+        :param required: List of required parameters.
+        """
+        self.name = name
+        self.description = description
+        self.properties = properties
+        self.required = required
```

Comment:
It seems these fields should be private (need an `_` suffix). If we want some of them to be accessible from outside we should create a `@property` so they are read-only.

Reply:
Done.